app/javascript/dashboard/routes/dashboard/settings/inbox/Settings.vue

```diff
@@ -438,7 +438,7 @@ export default {
       }
     },
     async updateAgents() {
-      const agentList = this.selectedAgents.map((el) => el.id);
+      const agentList = this.selectedAgents.map(el => el.id);
```

Comment:
I'm not sure whether this actually conforms to the eslint rules we have. This seems there is either a problem with the prettier config we have or there is some issue with the editor plugins you use. Could you please remove these?

Reply:
Some issue with `arrowParens` rule. I will add the rule `"arrowParens": "avoid"` in **.prettierrc** file